build_projects/dotnet-cli-build/PrepareTargets.cs

```diff
@@ -148,27 +148,38 @@ public static BuildTargetResult DownloadHostAndSharedFxArchives(BuildTargetConte
             var sharedFrameworkVersion = DependencyVersions.SharedFrameworkVersion;
             var sharedFrameworkChannel = DependencyVersions.SharedFrameworkChannel;
 
-            var combinedSharedHostAndFrameworkArchiveFile = c.BuildContext.Get<string>("CombinedFrameworkHostCompressedFile");
+            var combinedSharedHostAndFrameworkArchiveDownloadFile =
+                Path.Combine(Dirs.CoreSetupDownload, "combinedSharedHostAndFrameworkArchive");
+            var combinedSharedHostAndFrameworkArchiveDestinationFile = 
+                c.BuildContext.Get<string>("CombinedFrameworkHostCompressedFile");
 
-            Mkdirp(Path.GetDirectoryName(combinedSharedHostAndFrameworkArchiveFile));
+            Mkdirp(Path.GetDirectoryName(combinedSharedHostAndFrameworkArchiveDownloadFile));
+            Mkdirp(Path.GetDirectoryName(combinedSharedHostAndFrameworkArchiveDestinationFile));
 
-            AzurePublisher.DownloadFile(
-                AzurePublisher.CalculateArchiveBlob(
-                    combinedSharedHostAndFrameworkArchiveFile, 
-                    sharedFrameworkChannel, 
-                    sharedFrameworkVersion), 
-                combinedSharedHostAndFrameworkArchiveFile).Wait();
-
-            // Unpack the combined archive to shared framework publish directory
-            Rmdir(Dirs.SharedFrameworkPublish);
-            Mkdirp(Dirs.SharedFrameworkPublish);
-            if(CurrentPlatform.IsWindows)
+            if ( ! File.Exists(combinedSharedHostAndFrameworkArchiveDownloadFile))
             {
-                ZipFile.ExtractToDirectory(combinedSharedHostAndFrameworkArchiveFile, Dirs.SharedFrameworkPublish);
-            }
-            else
-            {
-                Exec("tar", "xf", combinedSharedHostAndFrameworkArchiveFile, "-C", Dirs.SharedFrameworkPublish);
+                AzurePublisher.DownloadFile(
+                    AzurePublisher.CalculateArchiveBlob(
+                        combinedSharedHostAndFrameworkArchiveDestinationFile,
+                        sharedFrameworkChannel,
+                        sharedFrameworkVersion),
+                    combinedSharedHostAndFrameworkArchiveDownloadFile).Wait();
+
+                File.Copy(combinedSharedHostAndFrameworkArchiveDownloadFile,
```

Comment:
Why do we copy the file?  Why not just unzip it from the place we downloaded it to?